File: leetcode/Easy/1365. How Many Numbers Are Smaller Than the Current Number/solution.py
class Solution:
    def smallerNumbersThanCurrent(self, nums: List[int]) -> List[int]:
        res = []
        for val in nums:
            count = 0
            for val2 in nums:
                if val > val2:
                    count += 1
            res.append(count)
        return res

File: leetcode/Easy/1365. How Many Numbers Are Smaller Than the Current Number/solution.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var smallerNumbersThanCurrent = function (nums) {
  let res = [];
  for (let val of nums) {
    let count = 0;
    for (let val2 of nums) {
      if (val > val2) count += 1;
    }
    res.push(count);
  }

  return res;
};
